src/constants/addresses.ts

```diff
@@ -149,6 +140,21 @@ export const BALANCER_VAULT_ADDRESSSES = {
   [NetworkId.MAINNET]: "0xba12222222228d8ba445958a75a0704d566bf2c8",
 };
 
+export const RANGE_OPERATOR_ADDRESSES = {
+  [NetworkId.MAINNET]: "",
+  [NetworkId.TESTNET_GOERLI]: "0x0E37549093CB786C2584eceCb0A96497fCD2aBA0",
+};
+
+export const RANGE_ADDRESSES = {
+  [NetworkId.MAINNET]: "",
+  [NetworkId.TESTNET_GOERLI]: "0xF4D2b752F1936fa69b98ab49BFD92EaEBdE4D266",
+};
+
+export const RANGE_PRICE_ADDRESSES = {
+  [NetworkId.MAINNET]: "",
+  [NetworkId.TESTNET_GOERLI]: "0x60421d7FD7AcD732384b1AB157F81a81E65296Ec",
+};
+
```

Comment:
One other minor thing. there will prob be new bond contracts as well. (at least the tellers). we'll need to update those alongside these so that the bonds appear in the bonds view

Reply:
updated in https://github.com/OlympusDAO/olympus-frontend/pull/2434: https://github.com/OlympusDAO/olympus-frontend/pull/2434/commits/a914ab3f9a47cffd1d586e6c792e92b8aae4211c

Reply:
this made some changes to abis & types